src_c/doc/rect_doc.h

```diff
@@ -21,6 +21,8 @@
 #define DOC_RECTCOLLIDERECT "colliderect(Rect) -> bool\ntest if two rectangles overlap"
 #define DOC_RECTCOLLIDELIST "collidelist(list) -> index\ntest if one rectangle in a list intersects"
 #define DOC_RECTCOLLIDELISTALL "collidelistall(list) -> indices\ntest if all rectangles in a list intersect"
+#define DOC_RECTCOLLIDEOBJECTS "collideobjects(rect_list) -> object\ncollideobjects(obj_list, key=func) -> object\ntest if an objects in a list intersects"
```

Comment:
> test if an objects in a list intersects

Should this be 'object' here, or 'any' ? I think 'any' actually.

Reply:
not sure, since I'm not a native English speaker I need some help: the method does the same as collidelist with the difference, that the list may contain objects and the object itself is returned instead of an index. Its like a FirstOrDefault filter. Could be worded like:

1. test if one object in a list intersects
2. test if any object in a list intersects

Both sound good to me, but may not to a native speaker.

Reply:
changing as suggested